internal/telemetry/telemetry.go

```diff
@@ -61,16 +63,30 @@ func Start() {
 	const deviceSalt = "64ee464f-9450-4b14-8d9c-014c0012ac1a"
 	deviceID, _ = machineid.ProtectedID(deviceSalt)
 
-	username := os.Getenv(envir.GitHubUsername)
-	if username != "" {
+	started = true
+}
+
+func userID(ctx context.Context) string {
+	tok, err := identity.Get().GenSession(ctx)
```

Comment:
Is there a way to never refresh the token here and in `orgID`?

Reply:
yeah, our auth library has a `GetSessions` which returns `refreshableTokenSource` which have a Peek() function. I'll switch to that.